--- tests/Fixtures/Standards/Typo3Update/Sniffs/Classname/InstantiationWithObjectManagerSniff/InputFileForIssues.php
+++ PHP_CodeSniffer
@@ -21,13 +21,13 @@

 // Not handled by this sniff, but StaticCallSniff, as this uses double colon.
 $this->objectManager->get(\Tx_Extbase_Command_HelpCommandController::class);
-$this->objectManager->get('\Tx_Extbase_Command_HelpCommandController');
-$this->objectManager->get('Tx_Extbase_Command_HelpCommandController');
+$this->objectManager->get('\TYPO3\CMS\Extbase\Command\HelpCommandController');
+$this->objectManager->get('\TYPO3\CMS\Extbase\Command\HelpCommandController');

 \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Extbase\Object\ObjectManager)
     // Not handled by this sniff, but StaticCallSniff, as this uses double colon.
     ->get(\Tx_Extbase_Command_HelpCommandController::class);
 \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Extbase\Object\ObjectManager)
-    ->get('\Tx_Extbase_Command_HelpCommandController');
+    ->get('\TYPO3\CMS\Extbase\Command\HelpCommandController');
 \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Extbase\Object\ObjectManager)
-    ->get('Tx_Extbase_Command_HelpCommandController');
+    ->get('\TYPO3\CMS\Extbase\Command\HelpCommandController');
