--- tests/Fixtures/Standards/Typo3Update/Sniffs/Classname/PhpDocCommentSniff/InputFileForIssues.php
+++ PHP_CodeSniffer
@@ -22,27 +22,27 @@
 class InputFileForIssues
 {
     /**
-     * @var Tx_Extbase_Domain_Repository_CategoryRepository
+     * @var \TYPO3\CMS\Extbase\Domain\Repository\CategoryRepository
      * @inject
      */
     protected $someVar;

     /**
-     * @var \TYPO3\CMS\Extbase\Persistence\ObjectStorage<Tx_Extbase_Domain_Model_Category>
-     * @validate Tx_Extbase_Validation_Validator_NumberValidator(property="value")
+     * @var \TYPO3\CMS\Extbase\Persistence\ObjectStorage<\TYPO3\CMS\Extbase\Domain\Model\Category>
+     * @validate \TYPO3\CMS\Extbase\Validation\Validator\NumberValidator(property="value")
      */
     protected $someRelation;

     /**
-     * @param t3lib_div
+     * @param \TYPO3\CMS\Core\Utility\GeneralUtility
      * @param \TYPO3\CMS\Backend\Template\MediumDocumentTemplate
      * @param \TYPO3\CMS\Perm\Controller\PermissionAjaxController
      *
-     * @return Tx_Extbase_Configuration_Configurationmanager
+     * @return \TYPO3\CMS\Extbase\Configuration\ConfigurationManager
      */
     public function doSomething($something)
     {
-        /** @var t3lib_div $variable */
-        /** @var $variable t3lib_div */
+        /** @var \TYPO3\CMS\Core\Utility\GeneralUtility $variable */
+        /** @var $variable \TYPO3\CMS\Core\Utility\GeneralUtility */
     }
 }
